src/components/statline/statline.tsx

```diff
@@ -0,0 +1,19 @@
+
+
+type Props = {
+  labels: string[]
+  values: string[]
+}
+
+export const StatLine = ({labels, values}: Props) => {
+  return (
+    <div className="flex justify-center m-1">
+    {values.map((value,i ) => (
+       <div key={i} className="items-center justify-center m-1 w-1/4 bg-blue-300">
```

Comment:
One more background removal - 

```suggestion
       <div key={i} className="items-center justify-center m-1 w-1/4">
```